### PR TITLE
fix: console error behaviour with no arguments

### PR DESCRIPTION
**Pull request by SimonSiefke**
## Changes/Tasks

- [x] Changed code

## Motivation and Context

fixes https://github.com/react-static/react-static/issues/1508

## Types of changes

- [ ] Refactoring/add tests (refactoring or adding test which isn't a fix or add a feature)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have updated the documentation accordingly
- [ ] I have updated the CHANGELOG with a summary of my changes
- [ ] My changes have tests around them

### CODE REVIEW

File: packages/react-static/src/utils/binHelper.js [hunk 1]
```diff
@@ -114,8 +114,16 @@ ignoredExtensions.forEach(ext => {
   require.extensions[`.${ext}`] = () => {}
 })
 
-console.error = (err, ...rest) => {
-  console.log(new PrettyError().render(err), ...rest)
+console.error = (...args) => {
+  if(args.length === 0){
+    return console.log()
```
Comment (SleeplessByte) [inline packages/react-static/src/utils/binHelper.js:119]:
```suggestion
    return undefined
```

Empty should bail out.

File: packages/react-static/src/utils/binHelper.js [hunk 2]
```diff
@@ -114,8 +114,16 @@ ignoredExtensions.forEach(ext => {
   require.extensions[`.${ext}`] = () => {}
 })
 
-console.error = (err, ...rest) => {
-  console.log(new PrettyError().render(err), ...rest)
+console.error = (...args) => {
```
Comment (SleeplessByte) [inline packages/react-static/src/utils/binHelper.js:117]:
```suggestion
const originalConsoleError = console.error
console.error = (...args) => {
```

File: packages/react-static/src/utils/binHelper.js [hunk 3]
```diff
@@ -114,8 +114,16 @@ ignoredExtensions.forEach(ext => {
...
+  if(args.length === 0){
+    return console.log()
+  }
+  const [err, ...rest] = args
+  if(err){
+    console.log(new PrettyError().render(err), ...rest)
+  } else {
+    console.log(err, ...rest)
+  }
```
Comment (SleeplessByte) [inline packages/react-static/src/utils/binHelper.js:126]: I checked [`pretty-error`](https://github.com/AriaMinaei/pretty-error) and it really only handled errors.

```suggestion
  if(err instanceof Error){
    console.log(new PrettyError().render(err), ...rest)
    return
  }
  
  return originalConsoleError(err, ...rest)
```

**Review by SleeplessByte — COMMENTED** [review]
Can you also add the CHANGELOG.md entry?

**Comment by mnlfischer** [comment]
@SleeplessByte please merge if pr is ok.

**Review by SleeplessByte — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Thank you for your contribution @SimonSiefke. It was [just released](https://github.com/react-static/react-static/releases/tag/v7.5.0).

**Pull request merged**